pkg/controller/servicebindingrequest/sbrcontroller.go

```diff
@@ -110,18 +110,28 @@ func updateFunc(logger *log.Log) func(updateEvent event.UpdateEvent) bool {
 		isConfigMap := isOfKind(e.ObjectNew, "ConfigMap")
 
 		if isSecret || isConfigMap {
-			dataFieldsAreEqual, err := compareObjectFields(e.ObjectNew, e.ObjectOld, "data")
+			dataFieldsAreEqual, err := compareObjectFields(e.ObjectOld, e.ObjectNew, "data")
```

Comment:
Is there any reason for this change?

Reply:
same question/.

Reply:
No particular reason, I was using Old as the first parameter in the following functions